hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/leaky/LeakyCryptoTestsSuite.java

```diff
@@ -586,7 +586,7 @@ private HapiSpec createAnAccountWithEVMAddressAliasAndECKey() {
                                 CRYPTO_CREATE_WITH_ALIAS_AND_EVM_ADDRESS_ENABLED),
                         overridingTwo(
                                 LAZY_CREATION_ENABLED,
-                                FALSE_VALUE,
+                                TRUE_VALUE,
```

Comment:
Not sure why we are changing this to `true`? AFAICT, we are also providing an `adminKey`, the created account won't be a `hollow account`, so the lazy creation flag shouldn't be needed, right?

Reply:
After the changes in this PR Line 613 is treated as hollow account creation. Thats why i changed the flag.
Since the functionality for hollow account creation with crypto create is still there I thought it would be better to have this validation (that is not possible to create a hollow account with already used evm address).

Reply:
Ah, yes, makes sense